cmd/build.go

```diff
@@ -38,9 +38,21 @@ var buildCmd = &cobra.Command{
 			fmt.Println("Please specify the container image name using flag '--image' or '-i'")
 			os.Exit(-1)
 		}
-		if err := build.BuildPushDockerImage(Dockerfile, DockerImage, DockerContext, PushImage); err != nil {
-			fmt.Println(err)
-			os.Exit(-1)
+
+		if s2iBuild {
```

Comment:
There should be a check for `--push` option. Push doesn't make sense with s2i

Reply:
Yeah `--push` does not make any sense because it is happening anyway. Can we not just ignore it?

Because there is no point in erroring out or anything?

Reply:
We can't just silently ignore it. Kedge should tell the user that he is trying to do something that doesn't make sense.

I would error out, but showing a warning that `--push` flag will be ignored is also fine.

Reply:
Help message for `--push` should say that it is just for Docker build

Reply:
done!